contrib/setup-cms.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
```

Comment:
Can you also add `-u` (expansion of unset variable fails)?

Reply:
🤦 if someone had asked me, I would have been certain I had `-u`!